.github/workflows/backwards_compatibility_tests_workflow.yml

```diff
@@ -14,8 +14,8 @@ jobs:
     strategy:
       matrix:
         java: [ 11, 17 ]
-        bwc_version : [ "1.1.0", "1.2.4", "1.3.2", "2.0.0", "2.1.0", "2.2.0" ]
-        opensearch_version : [ "2.3.0-SNAPSHOT" ]
+        bwc_version : [ "2.0.0", "2.1.0", "2.2.0", "2.3.0", "2.4.0-SNAPSHOT" ]
```

Comment:
do we need to check against the 2.4 snapshot?

Reply:
Yes. Pls find the below link for explanation
https://github.com/opensearch-project/OpenSearch/issues/3615#issuecomment-1247544266